src/main/java/nextstep/courses/domain/session/Enrolment.java

```diff
@@ -19,6 +23,9 @@ public void addParticipant(int money, NsUser user) {
     }
 
     private void validate(int money) {
+        if (status.equals(status.FINISH)) {
```

Comment:
status 값을 비교하지 않고 메서드로 표현해보아요!

```suggestion
        if (this.status.isFinish(status)) {
```